h2o-algos/src/main/java/hex/gam/GAMModel.java

```diff
@@ -388,7 +403,7 @@ public int nclasses() {
     /** Names of levels for a categorical response column. */
     @Override
     public String[] classNames() {
-      if (_family == Family.fractionalbinomial) {
+      if (_family == Family.fractionalbinomial || _family == Family.quasibinomial) {
```

Comment:
Same here, quasibinomial could have a different domain than {0,1}.

Reply:
Changed that too.